CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
+- Added cookie consent notice popup as opt-out. ([#7792](https://github.com/brave/brave-ios/issues/7792))
+- Added “View Recently Closed Tabs” to the tab tray context menu. ([#7752](https://github.com/brave/brave-ios/issues/7752))
+- Added network level standard blocking for cosmetic filtering. ([#7539](https://github.com/brave/brave-ios/issues/7539))
+- Updated the strings in the “restore your tabs” message (after experiencing a crash). ([#7695](https://github.com/brave/brave-ios/issues/7695))
+- Updated the URL bar context menu to show by long pressing on the search field as well the refresh icon in the URL bar. ([#7642](https://github.com/brave/brave-ios/issues/7642))
+- Updated New Tab Page favicon to display Brave icon instead of generic planet icon. ([#5357](https://github.com/brave/brave-ios/issues/5357))
+- Updated “Block Cookie Consent Notices” to be enabled by default. ([#7882](https://github.com/brave/brave-ios/issues/7882))
+- Updated logic for the Default Browser prompt on New Tab Page. ([#7860](https://github.com/brave/brave-ios/issues/7860))
+- Updated Sync Chain authentication logic to support Biometrics when enabling password sync or adding a new device. ([#7849](https://github.com/brave/brave-ios/issues/7849))
+- Updated that VPN Support Contact Form to append the issue category to the email subject field as well as body. ([#7828](https://github.com/brave/brave-ios/issues/7828))
+- Fixed issue with “Block Cookie Consent Notices” toggle switch being enabled when upgrading browser version. ([#7920](https://github.com/brave/brave-ios/issues/7920))
+- Fixed issue with Google OAuth window not auto-closing after successfully authenticating. ([#6966](https://github.com/brave/brave-ios/issues/6966))
+- Fixed URLs for "i.redd.it" posts on subreddits from forming incorrectly when using [old.reddit.com](http://old.reddit.com/) website redirect. ([#7873](https://github.com/brave/brave-ios/issues/7873))
+- Fixed issue allowing users to bypass the Browser Lock window using the “Cancel” button. ([#7893](https://github.com/brave/brave-ios/issues/7893))
+- Fixed “Brave can now be set as your default browser…” prompt to not be shown 14 days after installation. ([#7977](https://github.com/brave/brave-ios/issues/7977))
+- Fixed crash when migrating with very many tabs open. ([#7857](https://github.com/brave/brave-ios/issues/7857))
+- Fixed crash when viewing an advertisement in the Brave News feed after joining Brave Rewards. ([#7837](https://github.com/brave/brave-ios/issues/7837))
+- Fixed Tabs Bar to update properly when opening bulk bookmarks. ([#7836](https://github.com/brave/brave-ios/issues/7836))
+- Fixed restored but unopened tabs from being added to “Recently Closed Tabs” page. ([#7832](https://github.com/brave/brave-ios/issues/7832))
+- Fixed tabs to close properly with window.close(). ([#7796](https://github.com/brave/brave-ios/issues/7796))
```

Comment:
Can probably remove this one from the notes. I don't most people will understand the above. Maybe we can change it to something as follows:

```
Fixed tabs not being closed correctly in certain cases.
```